backend/README.md

```diff
@@ -160,7 +160,7 @@ npm install -g ts-node nodemon
 Then, run the watcher:
 
 ```
-nodemon src/index.ts --ignore cache/ --ignore pools.json
```

Comment:
We don't download this file anymore for a long time now, so let's remove it from the example